hardware/raspberrypi/bootmodes/net_tutorial.md

```diff
@@ -1,24 +1,20 @@
 # Network boot your Raspberry Pi
 
-This tutorial is written to explain how to set up a simple DHCP/TFTP server which will allow you to boot a Raspberry Pi 3 from the network. The tutorial assumes that you have an existing home network, and that you want to use a Raspberry Pi for the **server**. You will need a second Raspberry Pi 3 as a **client** to be booted. Only one SD card is needed because the client will be booted from the server after the initial client configuration.
+This tutorial is written to explain how to set up a simple DHCP/TFTP server which will allow you to boot a Raspberry Pi 3 or 4 from the network. The tutorial assumes that you have an existing home network, and that you want to use a Raspberry Pi for the **server**. You will need a second Raspberry Pi 3 as a **client** to be booted. Only one SD card is needed because the client will be booted from the server after the initial client configuration.
```

Comment:
I guess "a second Raspberry Pi 3" should be "a second Raspberry Pi 3 or Raspberry Pi 4" ? (and perhaps "second" should be "separate"?)

Reply:
> I guess "a second Raspberry Pi 3" should be "a second Raspberry Pi 3 or Raspberry Pi 4" ? (and perhaps "second" should be "separate"?)

Good spot - fixed.